config/locales/hello.es.yml

```diff
@@ -10,63 +10,59 @@ es:
 
     entities:
       sign_out:
         success: "Sesión finalizada!"
       unlink_access:
-        success: "Dispositivo ha sido desconectado satisfactoriamente de tu cuenta"
+        success: "Dispositivo desconectado satisfactoriamente de tu cuenta"
         alert: "Hubo un error al desconectar este dispositivo"
       session_switch:
-        # TODO: to Spanish
-        success: "Switched Accounts Successfully!"
+        success: "Has cambiado de cuentas satisfactoriamente!"
       session_forget:
-        # TODO: to Spanish
-        success: "Forgotten"
+        success: "Olvidó su contraseña"
       sign_up:
         success: "Tu cuenta ha sido creada correctamente"
         error:
-          one: "1 error ocurrió tratando de crear tu cuenta"
-          other: "%{count} errores ocurrieron tratando de crear tu cuenta"
+          one: "Ha ocurrido un error tratando de crear tu cuenta"
+          other: "Han ocurrido %{count} errores intentando crear tu cuenta"
       sign_in:
         success: "Sesión iniciada correctamente"
         error:
-          one: "1 error ocurrió tratando de inciar sesión"
-          other: "%{count} errores ocurrieron tratando de iniciar sesión"
+          one: "Ha ocurrido un error intentando iniciar sesión"
+          other: "Han ocurrido %{count} errores intentando iniciar sesión"
       forgot_password:
-        # TODO: adapt locale to Spanish
-        # success: "Hemos enviado un correo con instrucciones de como resetar tu contraseña"
-        success: "To get back into your account, follow the instructions we've sent to your <strong>\"%{login}\"</strong> email address."
+        success: "Hemos enviado un correo con instrucciones de como reestablecer tu contraseña a la dirección <strong>\"%{login}\"</strong>"
         error:
-          one: "1 error ocurrió tratando de encontrar tus credenciales"
-          other: "%{count} errores ocurrieron tratando de encontrar tus credenciales"
+          one: "Ha ocurrido un error intentado hallar tus credenciales"
+          other: "Han ocurrido %{count} errores intentando hallar tus credenciales"
       reset_password:
-        success: "Has reseteado tu contraseña satisfactoriamente"
-        alert: "Este enlace ha expirado, por favor pide un nuevo enlace"
+        success: "Tu contraseña ha sido restablecida satisfactoriamente"
+        alert: "Este enlace ha expirado, solicite uno nuevo por favor"
         error:
-          one: "1 error ocurrió mientras reseteabamos tu contraseña"
-          other: "%{count} errores ocurrieron mientras reseteabamos tu contraseña"
+          one: "Ha ocurrido un error mientras reestablecíamos su contraseña"
+          other: "Han ocurrido %{count} errores mientras reestablecíamos su contraseña"
       send_confirmation_email:
-        success: "Hemos enviado un correo de confirmación a %{email}"
+        success: "Se ha enviado un correo de confirmación a la direción %{email}"
```

Comment:
Let me start saying that IMO the previous version is slightly better, because I think that _Hemos enviado un correo de confirmación a bob@example.com_ is more personal than _Se ha enviado un correo de confirmación a la direción bob@example.com_.

As a side note **direción** contains a typo.

Also you can see that _a la dirección_ is redundant in this context, because the reader can infer that _bob@example.com_ is an email (_dirección de correo electrónico_).